conformance/vfs_assignement_test.go

```diff
@@ -0,0 +1,189 @@
+package conformance
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	. "github.com/onsi/gomega/gstruct"
+	netattdefv1 "github.com/openshift/sriov-network-operator/pkg/apis/k8s/v1"
+	sriovv1 "github.com/openshift/sriov-network-operator/pkg/apis/sriovnetwork/v1"
+	"github.com/openshift/sriov-tests/pkg/util/cluster"
+	"github.com/openshift/sriov-tests/pkg/util/execute"
+	"github.com/openshift/sriov-tests/pkg/util/namespaces"
+	v1core "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/pointer"
+	runtimeclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var _ = Describe("pod", func() {
+	var sriovInfos *cluster.EnabledNodes
+	execute.BeforeAll(func() {
+		var err error
+		sriovInfos, err = cluster.DiscoverSriov(clients, operatorNamespace)
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	var _ = Describe("Configuration", func() {
+		var nodePolicyConfig *sriovv1.SriovNetworkNodePolicy
+		var networkConfig *sriovv1.SriovNetwork
+		var testNode string
+
+		BeforeEach(func() {
+			testNode = sriovInfos.Nodes[0]
+			intf, err := sriovInfos.FindOneSriovDevice(testNode)
+			Expect(err).ToNot(HaveOccurred(), "No SR-IOV supported devices detected")
+			Expect(intf.TotalVfs).To(BeNumerically(">", 2), "Cluster has less than 2 VFs available.")
+
+			nodePolicyConfig = &sriovv1.SriovNetworkNodePolicy{
+				ObjectMeta: metav1.ObjectMeta{
+					GenerateName: "testpolicy",
```

Comment:
Can you give the policy a more descriptive name (like vfreleasedpolicy or something)?

Reply:
Done